x-pack/test_serverless/functional/test_suites/common/discover/x_pack/reporting.ts

```diff
@@ -103,7 +105,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.discover.selectIndexPattern('ecommerce');
       });
 
-      it('generates a report with single timefilter', async () => {
+      xit('generates a report with single timefilter', async () => {
```

Comment:
I guess this change will be undo, or any reason why this shouldn't run in serverless?

Reply:
Good catch - unskipped sorry and thanks!

Reply:
👍 thx!